farming_assistant/lib/APIs/user-related-apis.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+
+import '../models/user.dart';
+import '../utils/config.dart';
+
+Future<User> loginAPI(User user) async {
+  final uri = Uri.parse('${APIConfig.baseURI}/users/auth');
+
+  try {
+    final response = await http.post(
+      uri,
+      headers: {
+        'Content-Type': 'application/json',
+        //Login does not require an authorization header
```

Comment:
no need for this comm(se intelege de la sine)

Reply:
Outdated. Removed